apps/sms/test/marionette/lib/messages.js

```diff
@@ -51,43 +42,17 @@ var ConversationAccessor = require('./views/conversation/accessors');
 
         Composer: new ComposerAccessor(client),
```

Comment:
So before it was really NewMessage, but now it's really the "composer" part; shouldn't this be a property in NewMessage and Conversation instead ?